doc/api/net.md

```diff
@@ -137,6 +137,8 @@ The optional `callback` will be called once the `'close'` event occurs. Unlike
 that event, it will be called with an Error as its only argument if the server
 was not open when it was closed.
 
+Returns `server`.
```

Comment:
In various other parts of the docs (randomly selected example: https://nodejs.org/dist/latest-v8.x/docs/api/crypto.html#crypto_certificate_exportchallenge_spkac) the `Returns` is listed as a bullet point along with the expected arguments. For consistency, this should do the same.